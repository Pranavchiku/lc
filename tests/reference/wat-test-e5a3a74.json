{
    "basename": "wat-test-e5a3a74",
    "cmd": "lc --no-color --show-wat {infile} -extra-arg=\"\"",
    "infile": "tests/test.cpp",
    "infile_hash": "9e5ef4c37b7e9e1f13f0082ce65459c1e596430887537eb8fd656b77",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "wat-test-e5a3a74.stdout",
    "stdout_hash": "0e5c2efeca9f0626ad16508514a0a294d50bdef4a5b4763a15ad8bff",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}