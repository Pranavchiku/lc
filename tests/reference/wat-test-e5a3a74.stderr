Error while trying to load a compilation database:
Could not auto-detect compilation database for file "tests/test.cpp"
No compilation database found in $DIR/tests or any parent directory
fixed-compilation-database: Error while opening fixed database: No such file or directory
json-compilation-database: Error while opening JSON database: No such file or directory
Running without flags.
